src/Compilers/CSharp/Portable/Symbols/Symbol_Attributes.cs

```diff
@@ -295,6 +295,18 @@ internal virtual void PostDecodeWellKnownAttributes(ImmutableArray<CSharpAttribu
                 var attributeTypesBuilder = new NamedTypeSymbol[totalAttributesCount];
 
                 Binder.BindAttributeTypes(binders, attributesToBind, this, attributeTypesBuilder, diagnostics);
+                if (MessageID.IDS_FeatureGenericAttributes.GetFeatureAvailabilityDiagnosticInfo(compilation) is { } diagnosticInfo)
+                {
+                    for (var i = 0; i < totalAttributesCount; i++)
+                    {
+                        if (attributeTypesBuilder[i].IsGenericType)
+                        {
+                            diagnostics.Add(diagnosticInfo, attributesToBind[i].Location);
+                            break;
```

Comment:
>break [](http://example.com/codeflow?start=28&length=5)

Why `break;` rather than reporting an error for each generic attribute?

Reply:
I felt like it reduces redundancy of the diagnostics, since the fix for all the diagnostics is the same (upgrade the language version). But I don't have a strong opinion about it. Perhaps if the user wants to fix the problem by changing to not use a generic attribute, it would be better for them if they had a diagnostic on each attribute.

Reply:
Removed the `break;`.